src/actions/components/components-info

```diff
@@ -0,0 +1 @@
+React components go here
```

Comment:
I would also delete the actions directory and put these directories inside src directly

Reply:
Got it. I'm on it